tests/test_end_to_end.py

```diff
@@ -515,7 +515,7 @@ def test_end_to_end_search_download_cop_cds(self):
         product = self.execute_search(
             *COP_CDS_SEARCH_ARGS, search_kwargs_dict=COP_CDS_SEARCH_KWARGS
         )
-        expected_filename = "{}.grib".format(product.properties["title"])
+        expected_filename = "{}".format(product.properties["title"])
```

Comment:
test fails because file is downloaded in the following path:
```
/tmp/tmpm3fh64ma/ERA5_SL_20210101_43c1acf7bd098051d8e7bacd633482acd605f2c1.grib  
```
It should have been 
```
/tmp/tmpm3fh64ma/ERA5_SL_20210101_43c1acf7bd098051d8e7bacd633482acd605f2c1/ERA5_SL_20210101_43c1acf7bd098051d8e7bacd633482acd605f2c1.grib  
```